tests/pivots/pivot_data.ts

```diff
@@ -2,555 +2,188 @@ import { toZone } from "../../src/helpers/index";
 
 export const pivotModelData = function (xc: string) {
   return {
-    version: 19,
+    version: 25,
```

Comment:
not strictly necessary, but I realized it after it was done so... :man_shrugging: